Classes/GlobalStateExplorers/DatabaseBrowser/FLEXTableContentViewController.m

```diff
@@ -0,0 +1,183 @@
+//
+//  PTTableContentViewController.m
+//  PTDatabaseReader
+//
+//  Created by Peng Tao on 15/11/23.
+//  Copyright © 2015年 Peng Tao. All rights reserved.
+//
+
+#import "FLEXTableContentViewController.h"
+#import "FLEXMultiColumnTableView.h"
+#import "FLEXWebViewController.h"
+
+
+@interface FLEXTableContentViewController ()<FLEXMultiColumnTableViewDataSource, FLEXMultiColumnTableViewDelegate>
+
+@property (nonatomic, strong)FLEXMultiColumnTableView *multiColumView;
+
+@end
+
+@implementation FLEXTableContentViewController
+
+- (instancetype)init
+{
+  self = [super init];
+  if (self) {
+    
+    CGRect rectStatus = [UIApplication sharedApplication].statusBarFrame;
+    CGFloat y = 64;
+    if (rectStatus.size.height == 0) {
```

Comment:
If iPhone become landscape,The status bar is hidden,The height of navigation bar is 32 points, So the y = 32.